src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let cloneObj = { ...state };
```

Comment:
better to avoid calling variables like `result` or explicitly write the type of data we store, like `Obj`.
```suggestion
  const states = [];
  let clone = { ...state };
```